JavaDirectory/src/test/java/com/bug/free/invention/api/seleniumTests/jobRolesPageTest.java

```diff
@@ -0,0 +1,62 @@
+package com.bug.free.invention.api.seleniumTests;
+import com.bug.free.invention.api.framework.FunctionalTest;
+import com.bug.free.invention.api.pages.jobRolesPage;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class jobRolesPageTest extends FunctionalTest {
```

Comment:
Classes should always be Capitalised
```suggestion
public class JobRolesPageTest extends FunctionalTest {
```